Automattic/jetpack
# Widgets: Social icons: Add multiple ccTLDs for Pinterest to support icon display

Fixes https://github.com/Automattic/jetpack/issues/8422

#### Changes proposed in this Pull Request:


* Adds multiple ccTLDs for Pinterest, to support showing Pinterest icon - addresses the issue on https://github.com/Automattic/jetpack/issues/8422

This list was obtained from another project mentioned [here](https://github.com/matomo-org/searchengine-and-social-list/pull/45). I couldn't see an official list of ccTLDs owned by Pinterest so far. Happy to update with more ccTLDs, as suggested.

#### Testing instructions:





* Fire up a Jetpack site
* Add the `Social icons (Jetpack)` widget
* Try adding Pinterest links with the domains list below.
* Ensure the Pinterest icon shows up when you view the live site

---

- pinterest.com
- pinterest.ca
- pinterest.ch
- pinterest.co.uk
- pinterest.de
- pinterest.dk
- pinterest.es
- pinterest.fr
- pinterest.ie
- pinterest.jp
- pinterest.nz
- pinterest.pt
- pinterest.se

---

#### Before

The image below shows all icons for all the Pinterest ccTLDs, in the order listed earlier. Notice that the Pinterest icon does not appear for all TLDs that are not `.com` 

<img width="750" alt="screenshot 2018-12-22 at 20 54 30" src="https://user-images.githubusercontent.com/18581859/50375962-d8abaa00-062b-11e9-8999-a112f9a81229.png">

#### After

The image below shows all icons for all the Pinterest ccTLDs, in the order listed earlier as well. Notice that all the Pinterest icons appear fine.

<img width="750" alt="screenshot 2018-12-22 at 20 53 32" src="https://user-images.githubusercontent.com/18581859/50375963-de08f480-062b-11e9-92b3-cf38a419046d.png">

#### Proposed changelog entry for your changes:


* I guess no changelog entry is required, but feel free to let me know if I can mention "Social media widget now supports icon display for more ccTLDs" or something more clear. I think it's a very minor change that does not require an explicit changelog entry.
